src/core_plugins/elasticsearch/lib/client_logger.js

```diff
@@ -0,0 +1,35 @@
+module.exports = function (server) {
+  return class ElasticsearchClientLogging {
+    // additional tags to differentiate connection
+    tags = [];
+
+    logQueries = false;
+
+    error(err) {
+      server.log(['error', 'elasticsearch'].concat(this.tags), err);
+    }
+
+    warning(message) {
+      server.log(['warning', 'elasticsearch'].concat(this.tags), message);
+    }
+
+    info() {}
+
+    debug() {}
```

Comment:
Why are we nooping info and debug?  If we don't intend to implement them, I'd prefer that we throw an error if they're invoked.

Or if we're not using them at all in our logging stuff, let's just not define them.